core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -119,7 +119,11 @@ public Key() {
 
   /**
    * Creates a key with the specified row, empty column family, empty column qualifier, empty column
-   * visibility, timestamp {@link Long#MAX_VALUE}, and delete marker false.
+   * visibility, timestamp {@link Long#MAX_VALUE}, and delete marker false.  This constructor creates
+   * a copy of row. 
+   * 
+   * To avoid copying, use {@link Key#Key(byte[] row, byte[] cf, byte[] cq, byte[] cv, 
```

Comment:
Do you want this to be another paragraph?  Is so will need to add `<p>` to beginning.

Reply:
<p>added, thanks